erpnext/accounts/doctype/payment_entry/payment_entry_list.js

```diff
@@ -1,12 +1,52 @@
 frappe.listview_settings['Payment Entry'] = {
 
-	onload: function(listview) {
-		listview.page.fields_dict.party_type.get_query = function() {
+	onload: function (listview) {
+		listview.page.fields_dict.party_type.get_query = function () {
 			return {
 				"filters": {
 					"name": ["in", Object.keys(frappe.boot.party_account_types)],
 				}
 			};
 		};
+		listview.page.add_actions_menu_item(__('Print Check'), () => {
+			const selected_docs = listview.get_checked_items();
+			const doctype = listview.doctype;
+			if (selected_docs.length > 0) {
+				for (let doc of selected_docs) {
+					if (doc.docstatus !== 0) {
+						frappe.throw(__("Cannot print checks for 'Submitted' or 'Cancelled' documents"));
+					}
+					if (doc.mode_of_payment !== "Check") {
+						frappe.throw(__("{0}: The payment mode should be 'Check'",
```

Comment:
Let's change the text to refer to the field and avoid confusion

```suggestion
						frappe.throw(__("{0}: The mode of payment should be 'Check'",
```